content/events/current.md

```diff
@@ -2,6 +2,18 @@
 title: "Upcoming Events"
 ---
 
-## The CROCK
+#### The CROCK
```

Comment:
```suggestion
# The CROCK
```

Why is this a second level heading?

Reply:
2nd level heading is intentional. I want the text to be smaller than the "Past Events" header that comes one later.

Reply:
To be honest I can't tell that this is actually smaller. Maybe you can change the title to "Events" and make two top level sections: "Upcoming" and "Past"?